trackpy/preprocessing.py

```diff
@@ -52,19 +124,52 @@ def bandpass(image, lshort, llong, threshold=None, truncate=4):
             threshold = 1
         else:
             threshold = 1/256.
-    boxcar = image.copy()
-    result = np.array(image, dtype=np.float)
-    for axis, (sigma, smoothing) in enumerate(zip(lshort, llong)):
-        if smoothing > 1:
-            uniform_filter1d(boxcar, 2*smoothing+1, axis, output=boxcar,
-                             mode='nearest', cval=0)
-        if sigma > 0:
-            correlate1d(result, gaussian_kernel(sigma, truncate), axis,
-                        output=result, mode='constant', cval=0.0)
-    result -= boxcar
+    background = boxcar(image, [x * 2 + 1 for x in llong])
+    result = lowpass(image, lshort, truncate)
+    result -= background
     return np.where(result > threshold, result, 0)
```

Comment:
This probably should be `>=` . When threshold is 1, you would expect that everything below 1 becomes zero, but 1 itself does not.